feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -100,9 +103,27 @@ internal class HomeViewModel @Inject constructor(
             is ErrorShowed -> hideError()
             is CameraPermissionStatusChanged -> changeCameraPermissionStatus(event)
             CameraPermissionAsked -> markCameraPermissionAsAsked()
+            is CameraEvent -> { handleCameraEvent(event) }
         }
     }
 
+    private fun handleCameraEvent(event: CameraEvent){
```

Comment:
I was wondering, what if we will move all camera handle methods to the separate Handler? The benefit that we will achieve allows us to keep such a potential huge HomeViewModel in a small size. What do you think?

Reply:
Sure, I planned to do it, currently this PR is a draft